linode/manager
# fix: [M3-8523] - Fix styling on Support Ticket Details page

## Description 📝
In dark mode especially, the Support Ticket Details page needed some attention. I kept the scope of these changes small because there is future UX work to improve the design of this page further, at which point we can also refactor.

## Changes  🔄
- Fix dark mode container colors so that the expandable replies are distinct from each other
- Adjust avatar padding to better align the avatar and prevent jumping, especially on resize to small viewports

## Target release date 🗓️
9/30/24 or following release

## Preview 📷

| Before | After |
| ------- | ------- |
| Prod: ![Screenshot 2024-09-20 at 7 37 42 AM](https://github.com/user-attachments/assets/6d5b08c1-1bff-434c-bd8c-7996e94ca720) Dev: ![Screenshot 2024-09-20 at 7 50 23 AM](https://github.com/user-attachments/assets/105b5a4b-2991-4b0c-8df3-fc7cec444982) <video src="https://github.com/user-attachments/assets/242a00e0-a02b-46df-88a2-4bac21c55a22"/> | ![Screenshot 2024-09-20 at 7 44 05 AM](https://github.com/user-attachments/assets/14f5bc17-f799-4795-a263-3637b36d7b38) <video src="https://github.com/user-attachments/assets/86aeb740-b6cb-4054-adcd-cba7374acb85"/> |


## How to test 🧪

### Prerequisites
(How to setup test environment)
- Have an open or closed support ticket with multiple responses or use the MSW.
- Turn on dark mode as the theme

### Reproduction steps
(How to reproduce the issue, if applicable)
- Go to a support ticket's details page and observe the lack of visible containers around each reply
- Change the viewport size and notice with mobile view, the avatar is too close to the container

### Verification steps
(How to verify changes)
- Confirm the above are fixed

## As an Author I have considered 🤔

*Check all that apply*
